index.html

```diff
@@ -124,7 +124,17 @@ <h2 class="header-text">Profesional Art Printing Data</h2>
               </ul>
             </div>
             <button class="pjt-button main-btn">See Project</button>
-          </div>
+            </div>
+            <div class="project-modal-container" id="modal-container">
+              <div class="project-modal-content">
+                <img
+              src="assets/images/menu-close-Icon.png"
+              id="close-modal-icon"
+              alt="close modal icon"
+            />
+              </div>
+            </div>
+          
           <div class="project-card">
             <h2 class="header-text">Profesional Art Printing Data</h2>
             <p class="description2-text">
```

Comment:
- [ ] From the project requirement, you should display all these elements dynamically using javascript so these element needs to be removed :relaxed:  I think all this project information should be stored in an array of objects and iterate over displaying them dynamically using javascript.
  - **Please note** I couldn't select many elements that need to be removed from your HTML code but please note that all the HTML code that creates each project card needs to be removed from your HTML code and displayed dynamically using javascript. :pray: